manifests/params.pp

```diff
@@ -5,7 +5,10 @@
 # [Remember: No empty lines between comments and class definition]
 class pam_access::params {
 
-  if ($::osfamily == 'RedHat') and (versioncmp($::operatingsystemrelease, '5.0') >= 0) {
+  if ($::osfamily == 'RedHat') and (
+    (versioncmp($::operatingsystemrelease, '5.0') >= 0) or
```

Comment:
(($::osfamily == 'RedHat') and (
    (versioncmp($::operatingsystemrelease, '5.0') >= 0)) or

Reply:
modified